orchestrator-salt/src/main/resources/salt/salt/nginx/restoressl.sls

```diff
@@ -0,0 +1,18 @@
+restore_last_ssl_conf:
+  module.run:
+    - name: file.restore_backup
+    - path: /etc/nginx/sites-enabled/ssl.conf
+    - backup_id: 0
+
+touch_ssl_conf_for_restart:
```

Comment:
Perhaps add a dependency on `restore_last_ssl_conf`.

Reply:
I couldn't figure out how can I do that

Reply:
Not sure, perhaps add something like:
```
    - require:
        - module: restore_last_ssl_conf
```